src/main/java/gregtech/common/metatileentities/multi/electric/MetaTileEntityFusionReactor.java

```diff
@@ -323,7 +324,7 @@ protected long getMaxVoltage() {
         @Override
         public void updateWorkable() {
             super.updateWorkable();
-            if (!isActive && heat > 0) {
+            if (!(isActive || isWorkingEnabled()) && heat > 0) {
```

Comment:
Would `!recipeMapWorkable.isWorking() && heat > 0` be preferred instead here, so heat is also lost when there is not enough energy as well?